### PR TITLE
Fix exopackage app lib deps bug

### PR DESCRIPTION
**Pull request by Piasy**

### CODE REVIEW

File: buildSrc/build.gradle [hunk 1]
```diff
@@ -74,7 +74,7 @@ dependencies {
 def siteUrl = 'https://github.com/OkBuilds/OkBuck'
 def gitUrl = 'https://github.com/OkBuilds/OkBuck.git'
 def publishGroup = 'com.github.okbuilds'
-def publishVersion = '2.0.0-alpha1'
+def publishVersion = '2.0.0-alpha2'
```
Comment (kageiit) [inline buildSrc/build.gradle]: we dont need to change version till we actually cut a release

File: buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy [hunk 2]
```diff
@@ -100,8 +100,8 @@ class AndroidAppTarget extends AndroidLibTarget {
             String last // can denote either module or configuration name
             boolean fullyQualified = false
 
-            if (exoPackageDep.contains(";")) {
```
Comment (kageiit) [inline buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy]: ';' is correct. This is how we separate dependencies which are specified like group:module.

Please change it back

File: buildSrc/build.gradle (hunk 1 above)
Comment (Piasy) [inline buildSrc/build.gradle]: got it, I just test it locally right now, I'll remove it.

File: buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy (hunk 2 above)
Comment (Piasy) [inline buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy]: but this doesn't work

File: build.gradle [hunk 3]
```diff
@@ -49,8 +51,9 @@ okbuck {
     appLibDependencies = [
             'app': [
                     'buck-android-support',
-                    'multidex',
-                    'libraries:javalibrary',
+                    'com.android.support:multidex',
+                    'libraries/javalibrary:main',
```
Comment (kageiit) [inline build.gradle]: What is this format?

File: buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy (hunk 2 above)
Comment (kageiit) [inline buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy]: Nvm you are right :)

File: build.gradle [hunk 4]
```diff
@@ -49,8 +51,9 @@ okbuck {
...
+                    'libraries/common',
```
Comment (kageiit) [inline build.gradle]: do you want to add :release at the end?

File: buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy [hunk 5]
```diff
@@ -100,14 +100,15 @@ class AndroidAppTarget extends AndroidLibTarget {
...
-                List<String> parts = exoPackageDep.split(";")
+            if (exoPackageDep.contains(":")) {
+                List<String> parts = exoPackageDep.split(":")
                 first = parts[0]
                 last = parts[1]
                 fullyQualified = true
             } else {
                 first = last = exoPackageDep
             }
+            println "\t" + exoPackageDep + ", " + first + ", " + last + ", " + fullyQualified
```
Comment (kageiit) [inline buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy]: please remove this debug statement before merging

Comment (Piasy) [inline buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy]: sorry that I've already merged it, I'll remove it in next commit before publish

**Pull request merged**